src/edusharing-server-mock.ts

```diff
@@ -268,6 +268,10 @@ export class EdusharingServer {
     delete this.custom['dataToken']
   }
 
+  deletePostContentApiUrl() {
```

Comment:
Do you find a way to combine `deleteDataToken` and `deletePostContentApiUrl()` so that you do not have so much duplicated code?

-> By the way `postContentApiUrl` can be `null` (then the editor is rendered in view mode)

Reply:
Created a more generic cypress task and server mock function to delete arbitrary properties in custom.